Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm

```diff
@@ -267,9 +267,9 @@ - (CFURLRef)_cfurl;
 
 static RetainPtr<NSDictionary> policyProperties(const SameSiteInfo& sameSiteInfo, NSURL *url)
 {
-    static NSURL *emptyURL = [[NSURL alloc] initWithString:@""];
+    static NeverDestroyed<RetainPtr<NSURL>> emptyURL = adoptNS([[NSURL alloc] initWithString:@""]);
```

Comment:
I wonder if we should have a single shared one of these returned by a helper function rather than a local one here. Should search for other `initWithString:@""`.

Reply:
Interesting. There is also `URL::emptyCFURL()` which constructs a NSURL internally. However, it doesn't use a static variable.